javascript/ql/src/Security/CWE-347/MissingJWTKeyVerification.qhelp

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd"> <qhelp>
+
+<overview>
+<p>
+Applications decoding JSON Web Token (JWT) may be misconfigured due to the <code>None</code> algorithm.
```

Comment:
Not sure about this suggestion, but token in its singular form reads oddly in this sentence.
```suggestion
Applications decoding JSON Web Tokens (JWT) may be misconfigured due to the <code>None</code> algorithm.
```

Reply:
The plural version seems is used often enough, and yes it sounds better :+1: